docker-compose.yml

```diff
@@ -75,6 +76,7 @@ services:
       # This image needs to run highly privileged in order to
       # orchestrate user runtimes safely
       - ${DOCKER_SOCK}:/var/run/docker.sock
+      - gateway-vol:/var/lib
```

Comment:
It might be worth double checking this change

Reply:
Oh, I think I just missed this, this was the case for several of the volumes, but I opted to leave them as they were instead. I will do the same for this.

Reply:
Removed in favor of the old one.